src/durableorchestrationcontext.ts

```diff
@@ -5,7 +5,7 @@ import { TokenSource } from "./tokensource";
  * Parameter data for orchestration bindings that can be used to schedule
  * function-based activities.
  */
-export class DurableOrchestrationContext {
+export class DurableOrchestrationContext<T> {

```

Comment:
So we're removing these, right? To avoid propagating the "type" of the orchestration to constrain the type of the Entity state?

Reply:
Yep, haven't had time to do it